libs/wire-api/src/Wire/API/User/Profile.hs

```diff
@@ -240,6 +242,20 @@ instance FromByteString ManagedBy where
       "scim" -> pure ManagedByScim
       x -> fail $ "Invalid ManagedBy value: " <> show x
 
+instance FromHttpApiData ManagedBy where
+  parseUrlPiece txt = case txt of
+    "wire" -> pure ManagedByWire
+    "scim" -> pure ManagedByScim
+    _ -> Left $ "Invalid ManagedBy value: " <> txt
+
+instance ToHttpApiData ManagedBy where
+  toUrlPiece = \case
+    ManagedByWire -> "wire"
+    ManagedByScim -> "scim"
```

Comment:
This seems a bit redundant, can't we re-use the To/FromByteString instances, in the implementation?

Reply:
@blackheaven does this work?